release/karaf/itests/switchyard/pom.xml

```diff
@@ -377,7 +377,7 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-deploy-plugin</artifactId>
                 <configuration>
-                    <skip>true</skip>
+                    <skip>false</skip>
```

Comment:
Since `false` is the default, maybe we could remove the `skip` config completely?